serverless/internal/storage/webstorage/fs.go

```diff
@@ -0,0 +1,290 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build wasm
+
+// Package webstorage provides a simple log storage implementation based on webstorage.
+// It only really makes sense for wasm targets.
```

Comment:
I think the word Javascript would be good to add into here somewhere. The Go ecosystem seems to be favouring doc view (over code view) more, and I can easily see some developers (i.e. me if I hadn't had certain non-work conversations in my life) not knowing quite what this implies. It would be easy to assume `webstorage` was talking about putting stuff in google cloud, or github, or something.

Reply:
JavaScript is the wrong thing I think (because Node, etc.), but I updated to call out browsers and the reason why though.

Reply:
I suggested that based on the import of `syscall/js` but the new docs look good.